src/test/test_utils.cpp

```diff
@@ -749,21 +749,27 @@ void EnsureServerModelDownloadFinishedWithTimeout(ovms::Server& server, int time
     while ((server.getModuleState(ovms::HF_MODEL_PULL_MODULE_NAME) != ovms::ModuleState::SHUTDOWN) &&
            (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::high_resolution_clock::now() - start).count() < timeoutSeconds)) {
     }
-    ASSERT_EQ(server.getModuleState(ovms::HF_MODEL_PULL_MODULE_NAME), ovms::ModuleState::SHUTDOWN) << "OVMS did not download model in allowed time:" << timeoutSeconds << "s. Check machine load and network load";
+    // Actual download case
+    if (timeoutSeconds > 1) {
```

Comment:
```suggestion
    if (timeoutSeconds > 0) {
```
?

Why we need such check? It negates the purpose of this function